### DISCUSSION TITLE
How to add style to property elements?

### DISCUSSION THREAD

**Discussion by srivatsan2607** [GrapesJS/grapesjs#4804] [Q&A] [answered]
Hi @artf ,
I have added custom fonts during initialization as a property in the general sector. I would like to add style to each font in the list of the property.
For example, in the drop-down of fonts, I would like to show the particular font with that font family like Arial will be in Arial format, etc.
May I know how to implement stylings to the particular values in a property?
Thanks in advance.

**Comment by rajevar** [comment, marked as answer]
Styling to dropdown values are not supported by HTML. To achieve what you are trying to do, you have to create a custom `stylemanager` type that get rendered for `text` components. To give the styling effect ( arial style in your case) you will have to use bootstrap dropdown.. https://jsfiddle.net/rajevar/5zjfr2x8/1/
@artf  has this jsfiddle for custom style type which helped me a lot. https://jsfiddle.net/artur_arseniev/y1mxv6p5/

**Reply by srivatsan2607** [reply]
Thank you @rajevar , the solution was perfect.
